BForBoss.sln.DotSettings

```diff
@@ -1,2 +1,4 @@
 ﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=Accruement/@EntryIndexedValue">True</s:Boolean>
```

Comment:
Accruement is a word ([link](https://www.vocabulary.com/dictionary/accruement)) but Rider doesn't seen to know that, so added it to our solution's dictionary